factor_report.php

```diff
@@ -51,21 +51,47 @@
 foreach ($authtypes as $authtype) {
     $row = array();
     $row[] = \html_writer::tag('b', $authtype);
+
+    // Manually handle Total users and logged in users.
+    $totalusercountsql = "SELECT COUNT(DISTINCT id)
+                            FROM {user}
+                           WHERE auth = ?";
```

Comment:
filter deleted? / suspended? probably same below

Reply:
also would this be better to move outside the foreach and use a group by auth? same below